docs/devcontainer/tutorial.ipynb

```diff
@@ -112,8 +112,8 @@
       "Number of equality constraint Jacobian evaluations   = 19\n",
```

Comment:
Are there real changes in this file? If not, reducing the changes here will minimize merge conflicts for the other tutorial PR #67

Reply:
You'll have merge conflicts in Jupyter notebooks always. I don't think anyone should be responsible for resolving those regularly. I would prefer many small notebooks isolated for each concept. If I have changes here that are just affecting output, then just re-run the notebook and use your changes